README.md

```diff
@@ -1 +1,54 @@
-# crunchy
\ No newline at end of file
+# crunchy
+P2P network crawler data cruncher for graph metrics
+
+
+# Running
+
+crunchy gets two command line parameters: an input file, and an output file.  Both are `.json` files.
+
+The input file is a sample generated by our zcash crawler.  Its format (a JSON-RPC response) corresponds to this:
+
+
+```
+jsonrpc: String,
```

Comment:
shouldn't there be some enclosing brackets for this to be valid JSON?

Reply:
Well, it's not really JSON, but basically a Rust struct.  That said, I put it between curly braces.